waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/FederatedHMSHandler.java

```diff
@@ -194,27 +181,26 @@ public void shutdown() {
 
   @Override
   @Loggable(value = Loggable.DEBUG, skipResult = true, name = INVOCATION_LOG_NAME)
-  public String getMetaConf(String key) throws MetaException, TException {
+  public String getMetaConf(String key) throws TException {
```

Comment:
We just had this same discussion in the other recent PR and I'll restate what I said there - these additional exception types come from the Hive interface that we are implementing. I know the additional exceptions all extend from TException but I'm pretty sure they are called out on the interface in order to act as a form of documentation so you know some of the more specific things that can go wrong when you all the method. I don't think we should remove them, we should stick with what Hive does since we're implementing their interface. So please put these all back.

Reply:
Okay, I put the exception types back :)